components/admin/dialogs/AccountPlus.vue

```diff
@@ -5,26 +5,55 @@
 
     <v-list>
       <v-list-item-group>
-        <v-list-item @click="excelDialogOpen">
+        <v-list-item @click="selectTest">
           <v-list-item-icon>
-            <v-icon v-text="excel.icon" />
+            <v-icon v-text="test.icon" />
           </v-list-item-icon>
-          <v-list-item-content v-text="excel.text" />
+          <v-list-item-content v-text="test.text" />
         </v-list-item>
-        <v-list-item @click="manualDialogOpen">
+        <v-list-item @click="selectProduction">
           <v-list-item-icon>
-            <v-icon v-text="manual.icon" />
+            <v-icon v-text="production.icon" />
           </v-list-item-icon>
-          <v-list-item-content v-text="manual.text" />
+          <v-list-item-content v-text="production.text" />
         </v-list-item>
       </v-list-item-group>
     </v-list>
 
-    <v-dialog v-model="isExcelDialogOpen" max-width="600px">
+    <v-dialog v-model="isSelectDialogOpen" max-width="600px">
       <v-card>
-        <v-card-title>{{ cardTitle }}</v-card-title>
+        <v-card-title>{{ isTestUser ? test.title : production.title }}</v-card-title>
         <v-card-subtitle>{{ authSubTitle1 }}</v-card-subtitle>
+
+        <v-container class="form__scroll">
```

Comment:
ここやけど無駄なパディングが入っているから消して欲しいです！
見た目としては画像のような感じでお願いします！
<img width="1438" alt="スクリーンショット 2022-03-11 8 58 49" src="https://user-images.githubusercontent.com/49640294/157775685-0eb072f1-c887-43c9-8df0-95bef17e1926.png">

```suggestion
        <v-container class="form__scroll pa-n3">
```
参考サイト：https://vuetifyjs.com/ja/styles/spacing/